webapp/src/main/java/org/apache/atlas/web/rest/TypesREST.java

```diff
@@ -459,6 +462,58 @@ public AtlasTypesDef createAtlasTypeDefs(final AtlasTypesDef typesDef) throws At
         }
     }
 
+    private AtlasBaseException checkIndexing(AtlasTypesDef typesDef) throws AtlasBaseException {
```

Comment:
Rename the method name `checkIndexing`. If we are checking if index is present, plz convey the same w.r.t method name.